7.1/alpine/tools.json

```diff
@@ -581,6 +581,18 @@
       },
       "test": "psecio-parse --version"
     },
+    {
+      "name": "roave-backward-compatibility-check",
+      "summary": "Tool to compare two revisions of a class API to check for BC breaks",
+      "website": "https://github.com/Roave/BackwardCompatibilityCheck",
+      "command": {
+        "phar-download": {
+          "phar": "https://github.com/Roave/BackwardCompatibilityCheck/releases/download/1.1.0/roave-backward-compatibility-check.phar",
+          "bin": "/usr/local/bin/roave-backward-compatibility-check"
```

Comment:
This is a bit simplistic, and will likely only work if there is already a semver-compliant tag such as `X.Y.Z`

Reply:
It'll do for now.

BTW the build failed because the command expects params right away (registered a default command), to I'll add `--version`, but you don't set version in there, it says `Console application`.

Reply:
Doesn't matter here, might change it in the future.